common/djangoapps/student/views.py

```diff
@@ -678,7 +680,108 @@ def dashboard(request):
+                cert_info(request.user, course.course_overview, course.mode)
+            ) == "completed"
+        }
+    ]
+
+    current_courses = [
+        {
+            "course_id": course.id,
+            "course_key": unicode(course.course_id),
+            "course_number": course.course_overview.number,
+            "course_name": course.course_overview.display_name_with_default,
+            "course_image_url": course.course_overview.course_image_url,
+            "course_university_about_section": course.course_overview.display_org_with_default,
+            "course_mode": course.mode,
+            "has_course_ended": course.course_overview.has_ended(),
+            "has_course_started": course.course_overview.has_started(),
+            "starts_within_5_days": course.course_overview.starts_within(days=5),
+            "start_date_is_still_default": course.course_overview.start_date_is_still_default,
+            "short_end_date": course.course_overview.end_datetime_text("SHORT_DATE"),
+            "short_start_date": course.course_overview.start_datetime_text("SHORT_DATE"),
+            "day_and_time": course.course_overview.start_datetime_text("DAY_AND_TIME"),
+            "share_config": _get_share_config(request, course.course_overview),
+            "show_courseware_link": (course.course_id in show_courseware_links_for),
+            "may_certify": course.course_overview.may_certify(),
+            "cert_status": cert_statuses.get(enrollment.course_id),
+            "can_unenroll": (not cert_statuses.get(enrollment.course_id)
+                             or cert_statuses.get(enrollment.course_id).get('can_unenroll')),
+            "credit_status": _credit_statuses(user, course_enrollments).get(enrollment.course_id),
+            "show_email_settings": (course.course_id in show_email_settings_for),
+            "course_mode_info": course_mode_info.get(course.course_id),
+            "show_refund_option": (course.course_id in show_refund_option_for),
+            "is_paid_course": (course.course_id in enrolled_courses_either_paid),
+            "is_course_blocked": (course.course_id in block_courses),
+            "course_verification_status": verify_status_by_course.get(course.course_id, {}),
+            "course_requirements": _get_courses_requirements(courses_requirements_not_met.get(course.course_id)),
+            "course_program_info": course_programs.get(unicode(course.course_id)),
+            "course_verified_certs": enrollment_mode_display(course.mode, verification_status, course.course_id),
+            "optout": unicode(course.course_overview.id) in course_optouts
+        }
+        for course in {
+            course for course in course_enrollments
+            if course.course_overview.get_course_status(
+                cert_info(request.user, course.course_overview, course.mode)
+            ) in ['active', 'processing']
+        }
+    ]
+
+    course_settings = {
```

Comment:
None of these are actually course-specific settings. Please choose a less-confusing name.